test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/ChatCompletionResponseMessage.cs

```diff
@@ -32,6 +32,6 @@ internal ChatCompletionResponseMessage(ChatCompletionResponseMessageRole role, s
         /// <summary> The contents of the message. </summary>
         public string Content { get; }
         /// <summary> The name and arguments of a function that should be called, as generated by the model. </summary>
-        public CreateChatCompletionResponseFunctionCall FunctionCall { get; }
+        public ChatCompletionResponseMessageFunctionCall FunctionCall { get; }
```

Comment:
This is an anonymous model: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/completions/models.tsp#L301